tests/unit/static/golden/api.trulens_eval.3.11.yaml

```diff
@@ -1361,6 +1361,7 @@ trulens_eval.TruVirtual:
   - trulens.core.app.App
   __class__: pydantic._internal._model_construction.ModelMetaclass
   attributes:
+    add_dataframe: builtins.function
```

Comment:
probably don't need this here since add_dataframe was introduced after 1.0

Reply:
fixed